plugins/kubernetes.js

```diff
@@ -0,0 +1,77 @@
+const Client = require("kubernetes-client").Client;
+const aw = require("awaitify-stream");
+
+let kc;
+
+const kubernetes = {
+    name: "Kubernetes",
+    init: (controller) => {
+        controller.addPluginExtension("kubernetes", kubernetes);
+    },
+    getK8s: () => {
+        const version = "1.13";
+        kc = new Client({ version });
+        return kc;
+    },
+    startWatch: (
```

Comment:
since there isn't a `stopWatch` method in the plugin lets rename this method just `watch`

Reply:
I think we should introduce a `stopWatch` function though, it would help for facilitating a graceful shutdown of the process.

Reply:
`watch` (or `startWatch`) returns a method to shutdown the process. `const stop = plugins.kubernetes.watch(); stop();` To have a `startWatch` and `stopWatch` we would need for there to be an object that has a scope for each watch.

Reply:
right, which means we have no way to stop this when receiving a termination signal, so it would be difficult for this process to shut down gracefully.

Reply:
We discussed the ability to start/stop multiple watches at the application-level rather than handling it in the plugin. The current implementation is sufficient for planned usages(_application logic to start watches on startup_), however we could rework the plugin in the future if greater control is necessary.